src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,97 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { NavLink, useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from './Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>();
```

Comment:
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```
you will not get an error when trying to map/filter etc. empty array, but you will be able to get read of `?` sign everywhere you want to do it (people?.map)